packages/mdc-line-ripple/foundation.js

```diff
@@ -80,11 +80,11 @@ class MDCLineRippleFoundation extends MDCFoundation {
 
   /**
    * Sets the transform origin given a user's click location.
```

Comment:
I think this should be: `setRippleCenter(xCoordinate)`

Because really "transform-origin" is an internal implementation detail to line-ripple CSS.

And the description should be: "Sets the center of the ripple animation to the given X coordinate"

...or something in that direction anyway.